shell.nix

```diff
@@ -20,7 +9,9 @@ pkgs.mkShell {
     clightning
     jq
   ];
+buildInputs = [
+   (import ./default.nix )
```

Comment:
Is this a build input or package?

Reply:
it is a buildinput and alternative for nix-build